spec/animatable_region_spec.coffee

```diff
@@ -5,15 +5,33 @@ class TestPage extends Page
   render: ->
 
 describe 'AnimatableRegion', ->
-  before ->
-    @region = new AnimatableRegion(el: 'body')
-
-  it 'transitions pages', (done) ->
-    @region.show(new TestPage(id: 'page-1'))
-    @region.show(new TestPage(id: 'page-2'))
 
+  delayedAssert = (assert) ->
     setTimeout ->
-      expect($('#page-1')).not.to.exist
-      expect($('#page-2')).to.exist
+      assert()
       done()
     , 1000
+
+  currentPage = new TestPage(id: 'page-1')
+  nextPage = new TestPage(id: 'page-2')
+
+  before ->
+    @region = new AnimatableRegion(el: 'body')
+    @region.show(currentPage)
+
+  context 'with no transitions', ->
```

Comment:
This sounds like transitions are disabled, which is actually not the case.
I think you mean "Before transitioning to a new page".